### ISSUE TITLE
React-Native StoryBook not working with Chromatic

### ISSUE THREAD

**Issue by NM-2000** [chromaui/chromatic-cli#571]
Does Chromatic integrate with React-Native StoryBook?

Please follow this Issue Link -> https://github.com/storybookjs/react-native/issues/287

**Comment by ShaunLloyd** [comment]
Hey @NM-2000 

Right now Chromatic only integrates with web-based technologies which means we can't support the current version of Storybook for React-Native.

However, i've heard of some folks using react native web to get around this limitation

**Comment by matthew-dean** [comment]
When might this be added? And if people are using React Native Web, is there a guide for this?

**Comment by JReinhold** [comment]
@matthew-dean 
React Native support is not on our roadmap right now as there's no simple way to grab screenshots from a native Android/iOS app, similar to how we get screenshots of web apps.

Danny Williams wrote an excellent guide on how to set up Storybook (and therefore also Chromatic) with React Native Web here: https://levelup.gitconnected.com/introducing-react-native-web-storybook-3c56d48a6508
He's also the core maintainer of the React Native Web Addon: https://storybook.js.org/addons/@storybook/addon-react-native-web

In short it aliases the React Native components to web elements instead and makes them renderable in a web browser.

It's important to note though that this is a community effort and not something we officially support.

**Comment by paulelliott** [comment]
Closed due to age. Please reopen if you feel this is still relevant for you.
